static/app/components/profiling/flamegraph/flamegraphPreview.tsx

```diff
@@ -234,15 +250,24 @@ export function computePreviewConfigView(
   // If we were able to find a frame that is likely the parent of the span,
   // we should bias towards that frame.
   if (defined(innerMostParentFrame)) {
-    depth = Math.min(depth, innerMostParentFrame.depth);
+    if (depth > innerMostParentFrame.depth) {
```

Comment:
I really really like when we do these details right because it makes for such a nicer user experience